GetFed/GetFed/Helpers/CustomValueFormatter.swift

```diff
@@ -11,12 +11,7 @@ import Charts
 
 public class CustomValueFormatter: NSObject, IValueFormatter {
     
-    private let measurementFormatter: MeasurementFormatter
-    
-    init(measurementFormatter: MeasurementFormatter) {
-        self.measurementFormatter = measurementFormatter
-        super.init()
-    }
+    private let measurementFormatter = MeasurementFormatter()
```

Comment:
You're still making a new formatter for every `CustomValueFormatter` instance, and a new `NumberFormatter` every time it's called. You can create a static property so that same instance is usable for all calls:
```swift
private let measurementFormatter: MeasurementFormatter = {
    let formatter = MeasurementFormatter()
    formatter.unitOptions = .providedUnit

    let numberFormatter = NumberFormatter()
    numberFormatter.numberStyle = .none
    
    formatter.numberFormatter = numberFormatter

    return formatter
}()

Reply:
Thanks @jshier. I've pushed up an example. 

In the FoodSearchViewController, the CustomValueFormatter is assigned to the 'valueFormatter' of the data set whenever a new data set is created (at each segue trigger). So with this change, the same formatter will be used at each triggered segue?

Reply:
The same underlying `MeasurementFormatter`, but separate instances of `CustomValueFormatter`.

Reply:
I see. That makes sense, I'll incorporate these changes. Thanks again!